### ISSUE TITLE
Cool cat but it doesn't work for this use case

### ISSUE THREAD

**Issue by khatibomar** [GuidoFe/bash-cat-with-cat#1]
so our cat is cool , 
but it's add cat shape to txt file in following case
`cat >> file.txt`

**Comment by khatibomar** [comment]
I don't know if it's a good idea to alias original cat to writeto :p

**Comment by khatibomar** [comment]
Okay so my original cat is overrided with this new cat , so I will alias this cat to catty :3

**Comment by GuidoFe** [comment]
@OmarElKhatibCS Thank you for the feedback! I knew of this problem, now I managed to fix it. The command will behave like the normal cat if it's used inside a pipe.

**Comment by khatibomar** [comment]
it can't be done just like echo the top of cat then  call original cat with arguments passed . This will fix all the problems in my opinion.

**Comment by GuidoFe** [comment]
@OmarElKhatibCS Sorry I don't understand your explanation. Does it work now?

**Comment by khatibomar** [comment]
Yup it works now , thanks <3
